app/src/components/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+function App(){
+    return (
+        <div className="container">
```

Comment:
https://ru.bem.info/methodology/css/ 
take a look at this technology for class naming

Reply:
Do you mean this one?
 https://ru.bem.info/methodology/naming-convention/#%D1%81%D1%82%D0%B8%D0%BB%D1%8C-react

Reply:
> https://ru.bem.info/methodology/css/
> take a look at this technology for class naming

Do you mean this one?
https://ru.bem.info/methodology/naming-convention/#%D1%81%D1%82%D0%B8%D0%BB%D1%8C-react